app/server/utils/user.py

```diff
@@ -638,7 +636,7 @@ def proccess_create_or_modify_user_request(
     user = create_transfer_account_user(
         first_name=first_name, last_name=last_name, preferred_language=preferred_language,
         phone=phone, email=email, public_serial_number=public_serial_number, uuid=uuid,
-        organisation=organisation,
+        organisation=organisation if organisation else g.active_organisation,
```

Comment:
This being missing was causing a pretty major bug. Maybe this is important enough to be a separate PR?  
Basically what was going on was that `get_transfer_account_for_organisation` would run later in the user-creation flow, but no account would come up because the account wouldn't belong to the org making the account.